AK/BitStream.h

```diff
@@ -129,9 +138,26 @@ class LittleEndianInputBitStream : public Stream {
         return bits == 0 ? 0 : max >> (digits - bits);
     }
 
+    ALWAYS_INLINE BufferType lsb_aligned_buffer() const
+    {
+        return m_bit_offset == bit_buffer_size ? 0 : m_bit_buffer >> m_bit_offset;
+    }
+
+    ALWAYS_INLINE bool is_aligned_to_byte_boundary() const { return m_bit_count % bits_per_byte == 0; }
+
+    MaybeOwned<Stream> m_stream;
+
+    BufferType m_bit_buffer { 0 };
+    u8 m_bit_offset { 0 };
```

Comment:
Also, these two names are quite confusing IMO.
Maybe `m_bits_read` and `m_bits_in_buffer`?

Reply:
`m_bits_in_buffer` sounds a bit needlessly vebose to me, and `m_bits_read` adds confusion IMO because it implies strictly reading. These are also pre-existing names, so I'm not super interested in changing them in this PR.

Reply:
> m_bits_read adds confusion IMO because it implies strictly reading

Tunnel vision got me here, this is indeed terrible.